js/civs.js

```diff
@@ -1,301 +1,396 @@
-                [ARBALESTER, HEAVY_CAV_ARCHER, LIGHT_CAVALRY, HUSSAR, CAMEL_RIDER, HEAVY_CAMEL_RIDER,
-                    ELITE_CANNON_GALLEON, SIEGE_RAM],
-                [THUMB_RING, PARTHIAN_TACTICS, HUSBANDRY, DRY_DOCK, SHIPWRIGHT, BRACER,
-                    ARCHITECTURE, GOLD_SHAFT_MINING]);
-            unique([TEUTONIC_KNIGHT, ELITE_TEUTONIC_KNIGHT, IRONCLAD, CRENELLATIONS]);
-            break;
-        case "Turks":
-            disable([],
-                [ARBALESTER, ELITE_SKIRMISHER, PIKEMAN, HALBERDIER, PALADIN, FAST_FIRE_SHIP, ONAGER,
-                    SIEGE_ONAGER],
-                [HERBAL_MEDICINE, ILLUMINATION, BLOCK_PRINTING, STONE_SHAFT_MINING, CROP_ROTATION,
-                    SIEGE_ENGINEERS]);
-            unique([JANISSARY, ELITE_JANISSARY, SIPAHI, ARTILLERY], MONK_PREFIX_AFRICAN);
-            break;
-        case "Vietnamese":
-            enable([], [BATTLE_ELEPHANT, ELITE_BATTLE_ELEPHANT, IMPERIAL_SKIRMISHER], []);
-            disable([],
-                [HAND_CANNONEER, HUSSAR, PALADIN, CAMEL_RIDER, HEAVY_CAMEL_RIDER, FAST_FIRE_SHIP, SIEGE_RAM,
-                    SIEGE_ONAGER, HEAVY_SCORPION,],
-                [PARTHIAN_TACTICS, SHIPWRIGHT, REDEMPTION, HERESY, FERVOR, BLAST_FURNACE,
-                    MASONRY, ARCHITECTURE, GOLD_SHAFT_MINING]);
-            unique([RATTAN_ARCHER, ELITE_RATTAN_ARCHER, CHATRAS, PAPER_MONEY], MONK_PREFIX_ASIAN);
-            break;
-        case "Vikings":
-            enable([], [LONGBOAT, ELITE_LONGBOAT], []);
-            disable([KEEP, BOMBARD_TOWER],
-                [HAND_CANNONEER, HEAVY_CAV_ARCHER, HALBERDIER, HUSSAR, CAMEL_RIDER, HEAVY_CAMEL_RIDER, PALADIN,
-                    FIRE_GALLEY, FIRE_SHIP, FAST_FIRE_SHIP, SIEGE_ONAGER, BOMBARD_CANNON],
-                [PARTHIAN_TACTICS, BLOODLINES, HUSBANDRY, SHIPWRIGHT, KEEP, BOMBARD_TOWER, REDEMPTION,
-                    HERBAL_MEDICINE, SANCTITY, ILLUMINATION, THEOCRACY, PLATE_BARDING_ARMOR,
-                    STONE_SHAFT_MINING, GUILDS]);
-            unique([BERSERK, ELITE_BERSERK, CHIEFTAINS, BERSERKERGANG]);
-            break;
-    }
+
+    let selectedCiv = civsConfig[name];
+
+    let enabled = selectedCiv.enabled;
+    let disabled = selectedCiv.disabled;
+    if (selectedCiv.disableHorses) disableHorses(tree);
+
+    enable(enabled.buildings || [], enabled.units || [], enabled.techs || []);
+    disable(disabled.buildings || [], disabled.units || [], disabled.techs || []);
+    unique(enabled.unique || [], selectedCiv.monkPrefix);
 }
+
+const civsConfig = {
+    Aztecs: {
+        disableHorses: true,
+        enabled: {
+            units: [EAGLE_SCOUT, EAGLE_WARRIOR, ELITE_EAGLE_WARRIOR],
+            unique: [JAGUAR_WARRIOR, ELITE_JAGUAR_WARRIOR, ATLATL, GARLAND_WARS]
+        },
+        disabled: {
+            buildings: [KEEP, BOMBARD_TOWER],
+            units: [HAND_CANNONEER, HALBERDIER, CANNON_GALLEON, ELITE_CANNON_GALLEON, HEAVY_DEMO_SHIP, GALLEON, HEAVY_SCORPION, BOMBARD_CANNON],
```

Comment:
Could you limit lines to around 120 characters?

Reply:
I changed the whole file to avoid line length issues. Better?

Reply:
Marvellous! Having everything on its own line should also lead to prettier diffs for future balance changes.